src/MarginTrading.Backend.Core/OvernightSwaps/OvernightSwapCalculation.cs

```diff
@@ -17,15 +17,18 @@ public class OvernightSwapCalculation : IOvernightSwapHistory, IOvernightSwapSta
 		public decimal Volume { get; set; }
 		public decimal Value { get; set; }
 		public decimal SwapRate { get; set; }
-		public List<string> OpenOrderIds { get; set; }
+		public string OpenOrderId { get; set; }
 		
 		public bool IsSuccess { get; set; }
 		public Exception Exception { get; set; }
 		
 		public static string GetKey(string accountId, string instrument, OrderDirection? direction) =>
 			$"{accountId}_{instrument ?? ""}_{direction?.ToString() ?? ""}";
 
-		public static OvernightSwapCalculation Create(IOvernightSwapState state)
+	    public static string GetKey(string accountId, string instrument, OrderDirection? direction,string orderId) =>
```

Comment:
Since OrderId is unique it can be used as key directly. But not very important )